CONTRIBUTING.md

```diff
@@ -5,15 +5,17 @@ First of all, thank you for contributing. It’s appreciated.
 1. Open a GitHub issue before doing significant amount of work.
 2. Clone the repo. If it was already cloned, then git pull to get the latest from master.
 3. Use a Linux or Mac computer. No Windows, sorry.
-4. Ensure Yarn is [installed](https://yarnpkg.com/lang/en/docs/install/).
-5. Run `make setup` before anything else, and wait.
+4. Ensure pnpm is [installed](https://pnpm.js.org/docs/en/installation.html).
+5. Run `pnpm recursive install` before anything else, and wait.
```

Comment:
Don't you want `pnpm recursive link` instead? It will prefer linking packages from the repo, instead of downloading them from the registry (it is kind of like `lerna boostrap` but more isolated)